Mention Tournament and Battlefield in game modes (#778)
Heegu-sama/Homm3BG: sections/game_modes.tex

\begin{multicols*}{2}

Each game of Heroes III is played using a \textbf{Scenario} from the Mission Book. 
The core game only allows you to choose scenarios for 1-3 players. For 4 or more players, expansions are required.\par
In addition to the scenario objectives, read the \pagelink{Scenario end}{general victory conditions} depending on the scenario type.\par 
There are several types of scenarios:

\subsection*{Clash}

> A fully competitive mode. Your opponents even control neutral units in your combat.

Reviewer: ```suggestion
A fully competitive mode.
Your opponents control Neutral Units in your Combat.
```

\subsection*{Cooperative}
A Cooperative mode for 2--3 players where everyone shares the same goal.

\subsection*{Tournament}
A 1 vs.~1 mode with different way of building Map and scoring. See the Tournament Book.

\begin{center}
  \transparent{0.3}{\includegraphics[width=\linewidth, keepaspectratio]{\art/black_market.png}}
\end{center}

\columnbreak

> In addition, the \textbf{Battlefield Expansion} introduces two 1 vs.~1 modes which don't use Scenarios (see the expansion's rulebook):

Reviewer: ```suggestion
In addition, the \textbf{Battlefield Expansion} introduces two focused 1 vs.~1 modes without Scenarios (see the expansion's rulebook):
```
And here I think it flows better without the relative clause.

\subsubsection*{Adventure}

> The Map exploration is replaced with Adventure Cards. It culminates in a battle on a big hexagonal battlefield.

Reviewer: ```suggestion
The Map exploration is replaced with Adventure Cards, culminating in a tactical combat on a large hexagonal battlefield.
```
How about this? I think tactics is the main selling point for this mode.

\subsubsection*{Skirmish}

> Skip the adventure, and jump straight into the battle.

Reviewer: ```suggestion
Skip the exploration phase and engage directly in tactical combat.
```
Also, this. I just don't like "jumping" in this context.

A Cooperative mode for 2--3 players where everyone shares the same goal.

\subsection*{Tournament}
A 1 vs.~1 mode with different way of building Map and scoring. See the Tournament Book.

\vspace*{\fill}
\begin{center}
  \transparent{0.3}{\includegraphics[width=\linewidth, keepaspectratio]{\art/black_market.png}}
\end{center}

\columnbreak

> The \textbf{Battlefield Expansion} adds two scenario-free 1 vs.~1 modes:

Author: I would agree with this rewording if the rewritten book provided the rule for these. But it doesn't yet

Reviewer: Make it as you prefer, my main intention was to improve the layout.

\subsection*{Adventure}

> Explore using Adventure Cards instead of the Map, culminating in a hex-battlefield battle.

Author: "Explore using Map" sounds wrong

\subsection*{Skirmish}

> Direct combat without exploration.

Author: I think this is too short. Are you saying the combat can be direct and indirect?
